elide-async/src/test/java/com/yahoo/elide/async/service/AsyncQueryCleanerThreadTest.java

```diff
@@ -59,14 +62,12 @@ public void testAsyncQueryCleanerThreadSet() {
     @Test
     public void testDeleteAsyncQuery() {
         cleanerThread.deleteAsyncQuery();
-
-        verify(asyncQueryDao, times(1)).deleteAsyncQueryAndResultCollection(anyString());
+        verify(asyncQueryDao, times(1)).deleteAsyncQueryAndResultCollection(any());
```

Comment:
Similar to my above comment - stronger tests would have a list of mock queries with different conditions of staleness - and we verify that:

1. the ones we expect to be canceled are canceled
2. the ones that we expect to be deleted are deleted
3. the ones we expect to be ignored are ignored.